src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.Bytes.cs

```diff
@@ -37,7 +37,7 @@ private void WriteBase64StringHelper(ReadOnlySpan<byte> utf8PropertyName, ReadOn
         /// Writes the property name and raw bytes value (as a Base64 encoded JSON string) as part of a name/value pair of a JSON object.
```

Comment:
Unrelated but inconsistent use of "Base64". Let's changes this to "base 64", like the rest of the overloads.